src/Accelerators/NNPA/Conversion/ONNXToZHigh/ONNXLegalityCheck.cpp

```diff
@@ -848,6 +849,10 @@ bool isSuitableForZDNN<ONNXConvOp>(ONNXConvOp op) {
   if (!isWOK)
     return false;
 
+  // Workaround for issue #1517 Accuracy issue  at C != 1, kH = 1, kW=1
```

Comment:
Can you just add a bit of comment in the source file, e.g.
```
// Currently disable the generation of Conv2D when parameters are xxx
// because of current issue #1517.  When fixed, please remove lit test yyy.
```

Reply:
Done. Thanks!